Vagrantfile

```diff
@@ -3,14 +3,6 @@
 
 require 'yaml'
 
-# Check to see if docker-compose plugin is already installed
-# If not, the plugin is installed and you then have to rerun `vagrant up`
-unless Vagrant.has_plugin?("vagrant-docker-compose")
-  system("vagrant plugin install vagrant-docker-compose")
-  puts "Dependencies installed. Please try the command again"
-  exit
-end
-
 # All Vagrant configuration is done below. The "2" in Vagrant.configure
```

Comment:
Everything looks great! I just noticed there are some comments that don't seem relevant.

Reply:
@ottogiron Yes, I have removed those comments in this pull request, since I will not be using the docker compose plugin anymore.